pkg/core/system.go

```diff
@@ -291,6 +294,17 @@ func deleteNamespaces(kc *kubectlClient, namespaces []string) error {
 	return nil
 }
 
+func deleteNamespaceResource(kc *kubectlClient, namespace string, resourceType string, name string) error {
+	fmt.Printf("Deleting %s/%s resource in %s\n", resourceType, name, namespace)
+	deleteLog, err := kc.kubeCtl.Exec([]string{"delete", "-n", namespace, resourceType, name})
+	if err != nil {
+		if !strings.Contains(deleteLog, "NotFound") {
```

Comment:
Now that conditional logic is increasing, is it time to write a unit test?

Reply:
We don't have any tests for `core.system`, I created #1010

Reply:
Fair enough. It's probably non-trivial.